packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -194,6 +194,11 @@ export interface IFluidDataStoreChannel extends
      */
     getGCData(): Promise<IGarbageCollectionData>;
 
+    /**
+     * After GC has run, called to notify this channel of routes that are used in it.
```

Comment:
It would be great for comments also reflect of what is expected form implementation